packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -552,8 +557,20 @@ export function getValidatorApi({
       const blockValueEngine = enginePayloadValue + gweiToWei(consensusBlockValueEngine); // Total block value is in wei
 
       let executionPayloadSource: ProducedBlockSource | null = null;
+      const shouldOverrideBuilder = fullBlock?.shouldOverrideBuilder ?? false;
 
-      if (fullBlock && blindedBlock) {
+      // handle the builder override case separately
+      if (shouldOverrideBuilder === true) {
+        executionPayloadSource = ProducedBlockSource.engine;
+        logger.verbose("Selected engine block as censorship suspected in builder blocks", {
```

Comment:
I noticed we are not logging anything at info level when a block is produced or published by beacon node, all logs are either debug/verbose or error.

Maybe something to consider as this logs are very relevant for operators and a lot of the time they just look at info logs.

Reply:
This log is very relevant it should be info for sure

Reply:
right, finalselection logs should be info, will update

Reply:
Do we also want to log publishing on the beacon node as info? (should probably be done in a separate PR)

Reply:
sure, i can add it to cleanup todos (which i will be trying to push before mainnet release)